app/models/vm_or_template/scanning.rb

```diff
@@ -8,6 +8,12 @@
 module VmOrTemplate::Scanning
   extend ActiveSupport::Concern
 
+  # Smartstate Analysis is unsupported by default.
+  # Subclasses need to override this method if they support SSA.
+  included do
+    supports_not :smartstate_analysis, :reason => _("Smartstate Analysis is not available for VM or Template.")
```

Comment:
So? Wouldn't a user like to know that the button is disabled because it's not supported for a specific type, as opposed to some other reason?